src/chromo/models/phojet.py

```diff
@@ -59,6 +62,26 @@ def khard(self):
     #                   self._lib.poevt1.phep[0:4, 5]))
 
 
+class Phojet193Event(PhojetEvent):
+    def _fill_initial_beam(self):
+        super()._fill_initial_beam()
+        # Phojet193 produces mothers with first index > second index
```

Comment:
Shouldn't that also be the case for Phojet191 and also the old Phojet?

Reply:
Yes, this should be added for other Phojets. I checked once again and found that the problem occurs in Photjet191 and Phojet112 as well.